<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Map_Making">
    <property name="title">
     <string>地图</string>
    </property>
    <addaction name="emptyMap"/>
    <addaction name="MainRoadBendMap"/>
    <addaction name="setSize"/>
   </widget>
   <widget class="QMenu" name="menusearch">
    <property name="title">
     <string>搜索</string>
    </property>
    <addaction name="dfs"/>
    <addaction name="bfs"/>
    <addaction name="aStar"/>
   </widget>
   <addaction name="Map_Making"/>
   <addaction name="menusearch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="emptyMap">
   <property name="text">
    <string>空图</string>
   </property>
  </action>
  <action name="dfs">
   <property name="text">
    <string>Dfs</string>
   </property>
  </action>
  <action name="bfs">
   <property name="text">
    <string>Bfs</string>
   </property>
  </action>
  <action name="MainRoadBendMap">
   <property name="text">
    <string>随机生成地图</string>
   </property>
  </action>
  <action name="setSize">
   <property name="text">
    <string>设置大小</string>
   </property>
  </action>
  <action name="aStar">
   <property name="text">
    <string>A-star</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
